tensorflow/python/framework/dtypes.py

```diff
@@ -215,6 +217,27 @@ def max(self):
       except:
         raise TypeError("Cannot find maximum value of %s." % self)
 
+  @property
+  def limits(self, clip_negative=None):
```

Comment:
Define the `limits` for computing the scale of dtype, according to [scikit-image dtype_limits](https://github.com/scikit-image/scikit-image/blob/master/skimage/util/dtype.py#L32)